### DISCUSSION TITLE
Maneira ética de concorrer a outras vagas estando empregado

### DISCUSSION THREAD

**Discussion by valkervieira** [frontendbr/forum#1665] [Discussão]
Na opinião de vocês, qual a maneira ideal de se comportar na busca de novas oportunidades estando empregado?

A pergunta é mais específica pra uma situação em que você se considera "bem empregado", em uma empresa tranquila com um bom salário, mas talvez uma nova oportunidade traga um desafio novo, uma stack que você gostaria de trabalhar, um salário ainda melhor, etc.

Eu penso que avisar com antecedência que está indo em uma entrevista ou fazendo um teste por exemplo pode passar a ideia de "deslealdade" e criar uma desconfiança do tipo "fulano ta querendo sair", o que pode não ser inteiramente verdade.

Como vocês lidam com essa situação?

**Comment by SibilaSabrina** [comment]
Olá! Penso que seria bacana avaliar quanto tempo a empresa atual teria para se organizar com sua saída. Seu atual cargo/função possibilita a contratação de outro profissional com alguma rapidez?! Se sim, penso que é possível ir em busca de outras oportunidades e no momento adequado anunciar a saída, caso não aí complica um pouco, e ter uma conversa franca com a pessoa responsável pelo teu trabalho talvez seja o mais indicado, assim eles podem ter condições de se organizar do mesmo modo =)
Fique bem!

**Comment by ViniciusmDias** [comment]
Sinceramente, acho muito difícil isso ser visto com bons olhos pela empresa, principalmente se você é uma pessoa essencial para ela.
O que você pode fazer é tentar procurar desafios e estimular projetos novos na sua própria empresa, se isso não funcionar, significa que está realmente na hora de partir.
Pelo menos assim você passa a impressão de ter tentado algo novo e não sai com aquele peso na consciência por estar procurando um emprego, visto que já ficou claro que você está em busca de desafios.
Espero ter ajudado! Abraços

**Comment by kavalcante** [comment]
Bom, vou dar minha visão como empregador.

Realmente, não é legal você falar que está buscando outro emprego ou indo em outras entrevistas, porque as vezes você nem sabe se é isso mesmo que você quer. 

Acho que o mais correto é, caso você passe nas entrevistas, considerar se você aceita receber uma contra proposta e apresentar isso pra empresa que te emprega hoje.

Se a empresa não puder cobrir, ou você não quiser. Entra em acordo com o tempo que você precisa ficar (para mim o tempo deve ser de 1 semana até no máximo 1 mês). E vida que segue.

Quem tem uma empresa sabe que isso é normal e se a pessoa ficar puta, é melhor mesmo que você tenha saído. Deixando claro que você pode ficar um tempo até outra pessoa entrar e não sair do dia pra noite, não tem porque ter problemas.

**Comment by valkervieira** [comment]
Valeu pela resposta galera. Não é uma situação que eu estou passando no momento mas quando eu vim pra minha empresa atual eu fiz as entrevistas às escondidas e pra esse ano eu tenho objetivos pessoais que provavelmente não vão estar lado a lado com os objetivos da empresa comigo, mas eu tenho um carinho enorme pela minha equipe, quando o momento chegar (se chegar) eu quero fazer diferente do que fiz no passado.

Mais uma vez obrigado pelas respostas!

**Comment by eliseumds** [comment]
Sempre mantenha as portas abertas, e continue entrevistando ao menos uma ou duas vezes por ano, só pra ver onde você está encaixado, se é hora de sair ou de pedir um aumento. Isso é normal e esperado de qualquer pessoa curiosa e ambiciosa.
